src/components/reusables/UserInfoBar/UserInfoBar.module.css

```diff
@@ -0,0 +1,44 @@
+@media only screen and (min-width: 576px) {
+  .userInfoBarMargin{
+    margin: 0 8px 0 8px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .userInfoBarMargin{
+    margin: 0 16px 0 16px;
+  }
+}
+
+@media only screen and (min-width: 992px) {
+  .userInfoBarMargin{
+    margin: 0 40px 0 40px;
+  }
+}
+
+.altinnLogo{
+  flex: 1;
+}
+
+.userInfoBar{
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 1rem;
+  background-color: #1EADF7;
+  display: flex;
+  min-width: 320px;
+  max-width: 1056px;
+}
+
+.userInfoContent{
+  flex: 1;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 13px;
+}
+
+.userText{ 
+  font-weight: 500;
+  margin: 0;
+  margin-right: 50px;
```

Comment:
50px distance between the icon and the text feels a bit excessive.
How about 10px instead? (see picture)
![image](https://user-images.githubusercontent.com/22012997/221183676-16549fde-cdd4-4547-9562-ba50c56fb0e9.png)